sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -241,15 +241,6 @@ internal class RadarApiClient(
                 val mocked = location.isFromMockProvider
                 params.putOpt("mocked", mocked)
             }
-            if (tripOptions != null) {
```

Comment:
@corypisano Are we 100% sure there will be no behavior changes if we stop sending this? Is it useful to send up for debugging?

Reply:
(will continue to send)